### PR TITLE
az, az-apim, az-group, az-storage, az-tag, az-upgrade: add Spanish translation

### PR DESCRIPTION
**Pull request by crixodia**
Added Spanish translation for the following pages:

- az-apim
- az,
- az-storage
- az-group
- az-tag
- az-upgrade

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — CHANGES_REQUESTED** [review]
Thanks for your contribution.

Note to maintainers: This PR can be reviewed for files other than az-storage, we can review it after #10947's merge.

**Comment by kant** [comment]
@kbdharun ok

File: pages.es/common/az-apim.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+> Administra los servicios de Azure API Management.
+> Parte de `az`.
+> Más información: <https://docs.microsoft.com/cli/azure/apim>.
+
+- Enumera las instancias del servicio API Management:
+
+`az apim list --resource-group {{grupo_de_recursos}}`
+
+- Crear una instancia de servicio de API Management:
```
Comment (kant) [inline pages.es/common/az-apim.md:11]: Crear --> Crea

File: pages.es/common/az-apim.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+`az apim create --name {{nombre}} --resource-group {{grupo_de_recursos}} --publisher-email {{email}} --publisher-name {{name}}`
+
+- Elimina una instancia de servicio de API Management:
+
+`az apim delete --name {{nombre}} --resource-group {{grupo_de_recursos}}`
+
+- Muestra detalles de una instancia de servicio de API Management:
```
Comment (kant) [inline pages.es/common/az-apim.md:19]: instancia de servicio --> instancia del servicio

**Review by kant — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Review by kant — COMMENTED** [review]
LGTM

**Pull request merged**